heuristic/local_search/reinsertion.py

```diff
@@ -0,0 +1,31 @@
+from numpy.random import RandomState
+
+from heuristic.classes import Solution, Problem
+from heuristic.repair_operators import greedy_insert
+
+
+def reinsertion(current: Solution, rnd_state: RandomState) -> Solution:
```

Comment:
`reinsertion` is not a verb (reinsert is better). Reinsert what? I'd suggest you use `opt_reinsert_customers` instead - that's a lot more informative!